includes/process-donation.php

```diff
@@ -1166,18 +1164,18 @@ function give_donation_form_validate_cc_zip( $zip = 0, $country_code = '' ) {
 	return apply_filters( 'give_is_zip_valid', $ret, $zip, $country_code );
 }
 
-
 /**
  * Validate donation amount and auto set correct donation level id on basis of amount.
  *
  * Note: If amount does not match to donation level amount then level id will be auto select to first match level id on basis of amount.
  *
  * @param array $valid_data List of Valid Data.
- * @param array $data       List of Posted Data.
  *
  * @return bool
  */
-function give_validate_donation_amount( $valid_data, $data ) {
+function give_validate_donation_amount( $valid_data ) {
+	$data = $_POST;
```

Comment:
@raftaar1191 Do not need to create `$data` param please use `$_POST` directly with validation

Reply:
Here we are updating the $data and then adding that to DB
That why I am adding $_POST value to $data variable 


1. https://github.com/WordImpress/Give/blob/release/2.0.2/includes/process-donation.php#L1188

2. https://github.com/WordImpress/Give/blob/release/2.0.2/includes/process-donation.php#L1205


I can update this function but I don't think it's needed. Let me know if you want to to updated it